spec/controllers/trainees_controller_spec.rb

```diff
@@ -1,6 +1,24 @@
 require "rails_helper"
 
 RSpec.describe TraineesController do
```

Comment:
I don't think we need these. They're spreading like a virus 🤣 We cover this in the feature spec so unless there is some logic in the controller that isn't being exercised by the feature spec I think these are redundant.

Reply:
Should we delete all these then?

Reply:
I would vote yes for that

Reply:
I'll remove the one I add here and we can clear the others out later